biztalk/core/gmsa-config.md

```diff
@@ -0,0 +1,84 @@
+---
+title: BizTalk Server - group managed service account (gMSA) | Microsoft Docs
+description: Configure and use BizTalk Server with group managed service account (gMSA)
+author: msjaydeep
+ms.author: jaah
+manager: dougeby
+ms.date: 01/03/2019
+ms.topic: config
+ms.prod: biztalk-server
+
+# optional metadata
+
+#ROBOTS:
+
+ms.reviewer: 
+ms.suite:
+ms.tgt_pltfrm:
+ms.assetid: 
+ms.custom: 
+---
+
+# Using Group Managed Service Account for BizTalk Server Features
+
+BizTalk 2020 Supports Group Managed Service Account. Detailed information about gMSA can be found [here](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) .
```

Comment:
remove /en-us/ from the link? so it will redirect to current language user is using.